### PR TITLE
kprf.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

**Comment by jeremyn** [comment]
This ruleset is missing a lot of subdomains, see [here](https://www.google.com/search?q=site%3A*.kprf.ru&ie=utf-8&oe=utf-8#q=site:*.kprf.ru+-site:www.kprf.ru+-site:www.zyuganov.kprf.ru+-site:ni.kprf.ru). Please add them.

**Comment by galeksandrp** [comment]
Added.

File: src/chrome/content/rules/kprf.ru.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+
+¹ mismatch
+² refused
+⁷ protocol error
+-->
+<ruleset name="kprf.ru">
+	<target host="kprf.ru" />
+	<target host="www.kprf.ru" />
+	<target host="rlist.kprf.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/kprf.ru.xml]: msk.kprf.ru

**Comment by J0WI** [comment]
Thanks!

**Pull request merged**